.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper
 install:
-  - git clone --branch=master https://github.com/WycliffeAssociates/kotlin-resource-container.git ../kotlin-resource-container
+  - git clone --branch=dev https://github.com/WycliffeAssociates/kotlin-resource-container.git ../kotlin-resource-container
   - git clone https://github.com/WycliffeAssociates/8woc2018-common.git ../8woc2018-common
```

Comment:
Maybe you should edit `init_common.sh` (and rename it to `init_sub_projects.sh`), so Travis can build in-flight features when kotlin-resource-container hasn't yet been merged.

...but I don't know whether any of this should happen in a zip file PR.

@sarabiaj might correct me on this idea though....

Reply:
This seems reasonable

I'm also open to revisiting the idea of putting all modules in one repo as well though (and adding into it Android too)

Reply:
@sarabiaj will talk to @danparisd about whether we need something different here, so this is resolved for the sake of this PR.